swap_meet/item.py

```diff
@@ -1,2 +1,37 @@
+import uuid
+
 class Item:
-    pass
\ No newline at end of file
+    def __init__(self, id = None, condition = 0):
+        if id is  None:
+            self.id =  uuid.uuid4().int 
+        else:
+            self.id = id 
+        
+        if isinstance(condition, (int, float)):
+            if 0 <= condition <= 5:
+                self.condition = condition 
+            else:
+                raise ValueError("Condition must be between 0 and 5")
+        else: 
+            raise TypeError("Condition must be a number")
+
+    
+    def get_category(self):
+        return self.__class__.__name__
+    
+    def __str__(self):
+        return f"An object of type Item with id {self.id}."
+
+    def condition_description(self):
+        if self.condition == 5:
```

Comment:
Explicitly writing the logic does give us the ultimate control over exactly under which situations a particular description should be used, but especially when we have a long list of chained checks that are very similar, we could try coming up with a way to represent it with data and use iteration. A basic example of this that would mirror your existing behavior could look like:
```python
# outside class definition
DESCRIPTIONS = (
    "Heavily Used",
    "Poor",
    "OK",
    "Good",
    "Excellent",
    "Mint",
)

# in Item class definition
    def condition_description(self):
        return DESCRIPTIONS[self.condition]
```
We would still want to think about the edge case behaviors (outside the expected range? non-integer conditions?), but we would have more general confidence that if this logic works for a particular condition, it's likely to work for all of them.